src/transformers/models/roberta/modeling_flax_roberta.py

```diff
@@ -160,243 +163,202 @@ class FlaxRobertaEmbedding(nn.Module):
 
     vocab_size: int
     hidden_size: int
-    kernel_init_scale: float = 0.2
-    emb_init: Callable[..., np.ndarray] = jax.nn.initializers.normal(stddev=kernel_init_scale)
+    initializer_range: float
     dtype: jnp.dtype = jnp.float32  # the dtype of the computation
 
-    @nn.compact
-    def __call__(self, inputs):
-        embedding = self.param("weight", self.emb_init, (self.vocab_size, self.hidden_size))
-        return jnp.take(embedding, inputs, axis=0)
+    def setup(self):
+        emb_init: Callable[..., np.ndarray] = jax.nn.initializers.normal(stddev=self.initializer_range)
+        self.embeddings = self.param("weight", emb_init, (self.vocab_size, self.hidden_size))
```

Comment:
```suggestion
        self.embeddings = self.param("weight", init, (self.vocab_size, self.hidden_size))
```
If previous suggestion is accepted.